### PR TITLE
Update protobuf to v21.5

### PR DESCRIPTION
**Pull request by sagmor**
Just updated the hashes from #125.

### CODE REVIEW

**Comment by sagmor** [comment]
CC @JohnTitor as I took over your change feel free to take it over if you want 👍 
CC @loeffel-io as you requested this (I'm blocked on envoyproxy/protoc-gen-validate/583 too 😅)

File: proto/private/BUILD.release [hunk 1]
```diff
@@ -54,7 +54,7 @@ proto_lang_toolchain(
 # Redirect everything else to the source
 redirect_targets = [
     "wrappers_proto",
-    "well_known_protos",
+    "well_known_types_proto",
```
Comment (comius) [inline proto/private/BUILD.release:57]: This seems to contain a two typos, it should be "well_known_type_protos".

To verify manually `blaze build //proto/private:all` should work.

**Comment by comius** [comment]
Thanks @sagmor!!!

**Comment by JohnTitor** [comment]
Thanks! I totally forgot to update my PR.

**Review by comius — APPROVED** [review]

**Comment by comius** [comment]
I also mirrored the files.

**Pull request merged**